rskj-core/src/integrationTest/java/co/rsk/cli/tools/CliToolsIntegrationTest.java

```diff
@@ -0,0 +1,599 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package co.rsk.cli.tools;
+
+import co.rsk.RskContext;
+import co.rsk.core.BlockDifficulty;
+import co.rsk.crypto.Keccak256;
+import co.rsk.trie.Trie;
+import co.rsk.util.HexUtils;
+import co.rsk.util.TestUtils;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.squareup.okhttp.*;
+import org.ethereum.core.Block;
+import org.ethereum.util.ByteUtil;
+import org.ethereum.util.FileUtil;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import javax.net.ssl.*;
+import java.io.*;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class CliToolsIntegrationTest {
+    private String projectPath;
+    private String buildLibsPath;
+    private String jarName;
+    private String databaseDir;
+    private String bloomsDbDir;
+    private final JsonNodeFactory jsonNodeFactory = JsonNodeFactory.instance;
+    private final int port = 9999;
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private String[] baseArgs;
+    private LinkedList<String> lnkListBaseArgs;
+    private String strBaseArgs;
+    private String integrationTestResourcesPath;
+    private String logbackXmlFile;
+    private String rskConfFile;
```

Comment:
Fields can be converted to a local variable

Reply:
Addressed